I work hard for the money.

gojamesgo85 opened the pull request:
Started adding cocktails from the W&G catalogue. Many more to go.

MatthewHunt84 commented on MetaCocktailsSwiftData/Model/Cocktails/Classics/LastWord.swift:
```diff
@@ -10,7 +10,7 @@ import SwiftUI
                         glasswareType: .stemmedGlassware,
                         ice: nil,
-                        author: "Frank Fogarty or the Detroit Athletic club",
+                        author: ["Frank Fogarty or the Detroit Athletic club"],
```
Was Frank Fogarty the bartender at the Detroit Athletic club who came up with it?

MatthewHunt84 commented on MetaCocktailsSwiftData/Model/Cocktails/Williams and Graham/SlutDragon.swift:
```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+
+var slutDragon = Cocktail(cocktailName: "Slut Dragon (Aka: Soul Bond)",
```
pick one lol, this was the soul bond but you can rename it now if you want

MatthewHunt84 commented on MetaCocktailsSwiftData/Model/Components/Ingredients/IngredientType.swift:
```diff
@@ -208,12 +209,13 @@ enum MeasurementUnit: String, Codable {
+    case nitroMuddled   = "Nitro muddled"
     case ml             = "ml"
-    case barspoon       = "Barspoon"
+    case barspoon       = "Bar-spoon"
```
Barspoon is a relatively inaccurate measurement that I think we should move away from.
Since we only use barspoon in 5 recipes right now, can we change those recipes into teaspoon measurements instead?

MatthewHunt84 commented on MetaCocktailsSwiftData/Model/SearchTags/Authors.swift:
```diff
@@ -0,0 +1,21 @@
+    case philWard          = "Phil Ward"
+    case jerryThomas       = "Jerry Thomas"
+    
+    
```
The author part of the model is a bit messy right now. Sometimes it's an author, sometimes its a bar, sometimes its a place. I think a better solution is to make Author a struct with a name, cocktailBar, and year. All optional. We don't need to use a place like New Orleans, if the author is unknown that's ok. 

Then we can make a presentationString variable on the struct as well that looks at what it has and returns the string we'll use on the cocktail page.
So depending on the author struct, calling Author.jamesMenkal.presentationString will return "James Menkal" or "James Menkal, Williams and Graham" or "James Menkal, Williams and Graham (2019)" depending on what information is available in the struct

MatthewHunt84 reviewed (changes requested):
Tons of work, great job. I have a couple of nitpicky things that could be addressed - but keep up the great work. I'll commit myself much more to UI stuff, just gotta learn a bit more about SwiftUI first

gojamesgo85 commented:
Changed everything you suggested.


Created Author struct with optionals.
Added GeometryReader bounds to RECIPEINGREDIENTSVIEW so everything fits on my shitty little iPhoneSE.
Added some more cocktails.
